docs/en/observability/cloud-monitoring/aws/ingest-aws-options.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[ingest-aws-options]]
+==  Ingest options
+
+++++
+<titleabbrev>Ingest options</titleabbrev>
+++++
+
+You have a number of options for ingesting data with AWS. The following table helps you identify which option best fits your needs:
+
+[cols="1,1,1,1,1",options="header"]
```

Comment:
I like this table, but it's confusing to me that it's in a different order than the list right above it.

Reply:
In the new structure, the page is organized differently. The list above the table has been removed.